pg_lakehouse/tests/table_config.rs

```diff
@@ -148,3 +153,52 @@ async fn test_preserve_casing(mut conn: PgConnection, tempdir: TempDir) -> Resul
 
     Ok(())
 }
+
+#[rstest]
+async fn test_preserve_casing_on_table_create(
+    mut conn: PgConnection,
+    tempdir: TempDir,
+) -> Result<()> {
+    let stored_batch = primitive_record_batch()?;
+    let parquet_path = tempdir.path().join("test_arrow_types.parquet");
+    let parquet_file = File::create(&parquet_path)?;
+
+    let mut writer = ArrowWriter::try_new(parquet_file, stored_batch.schema(), None).unwrap();
+    writer.write(&stored_batch)?;
+    writer.close()?;
+
+    primitive_setup_fdw_local_file_listing(parquet_path.as_path().to_str().unwrap(), "MyTable")
+        .execute(&mut conn);
+
+    format!(
+        "CREATE FOREIGN TABLE \"MyTable\" () SERVER parquet_server OPTIONS (files '{}', preserve_casing 'true')",
+        parquet_path.to_str().unwrap()
+    )
+        .execute(&mut conn);
+
+    match "SELECT foreign_table_name FROM information_schema.foreign_tables WHERE foreign_table_name='MyTable';".fetch_result::<ForeignTableInformationSchema>(&mut conn) {
```

Comment:
I don't think we need to introduce a new struct here. See the other tests for examples of how to collect `SELECT` results into a `Vec`.

Reply:
Sounds good! I'll get to this either tonight or tomorrow! I'm hiking today.

Reply:
should be good now :D